operators/src/engine/query_processor.rs

```diff
@@ -39,29 +30,13 @@ pub trait RasterQueryProcessor: Sync + Send {
     }
 }
 
-#[async_trait]
-impl<S, T> RasterQueryProcessor for S
-where
-    S: QueryProcessor<Output = RasterTile2D<T>> + Sync + Send,
-    T: Pixel,
-{
-    type RasterType = T;
-    async fn raster_query<'a>(
-        &'a self,
-        query: QueryRectangle,
-        ctx: &'a dyn QueryContext,
-    ) -> Result<BoxStream<'a, Result<RasterTile2D<Self::RasterType>>>> {
-        self.query(query, ctx).await
-    }
-}
-
```

Comment:
I guess we can still default implement it and just pass in a different Qrect

Reply:
why do we need it?

Reply:
We still have a common interface for querying stuff.

Reply:
but what is the benefit of using a common query method if both the input and the output types are different?